mitodl/ocw-studio
# first round of styling changes

#### Pre-Flight checklist


#### What are the relevant tickets?

closes #115 

#### What's this PR do?

This brings the site more in line with the mockups. It doesn't implement every single thing in he mockups (notably the different header style shown therein on the site pages) but it gets most of the site in shape.

- background colors
- text styling (Abdul recommended going with Google Roboto for our basic font)
- button styling
- adding content cards to wrap forms and so on

and a few more random enhancements besides.

#### How should this be manually tested?

check out the mocks, and then click around the site. Make sure that no functionality has been compromised anywhere. Then check that all butons are switched over, and that all the gray elements have been eliminated.



#### Screenshots (if appropriate)

![Screenshot from 2021-03-19 11-01-21](https://user-images.githubusercontent.com/6207644/111800567-78efe800-88a2-11eb-8b98-0f9a11c20e25.png)
![Screenshot from 2021-03-19 11-01-15](https://user-images.githubusercontent.com/6207644/111800568-79887e80-88a2-11eb-9e81-16a02c478a6d.png)
![Screenshot from 2021-03-19 11-01-11](https://user-images.githubusercontent.com/6207644/111800570-79887e80-88a2-11eb-8940-ebd93a2da5d4.png)
![Screenshot from 2021-03-19 11-01-05](https://user-images.githubusercontent.com/6207644/111800571-79887e80-88a2-11eb-9209-241675c4bd9b.png)
![Screenshot from 2021-03-19 11-00-59](https://user-images.githubusercontent.com/6207644/111800572-7a211500-88a2-11eb-9aaf-4055a75ded72.png)
![Screenshot from 2021-03-19 11-00-54](https://user-images.githubusercontent.com/6207644/111800573-7a211500-88a2-11eb-9ec4-9f6fdc9f8b66.png)
![Screenshot from 2021-03-19 11-00-50](https://user-images.githubusercontent.com/6207644/111800574-7a211500-88a2-11eb-8f5d-0418c141a339.png)
![Screenshot from 2021-03-19 11-00-40](https://user-images.githubusercontent.com/6207644/111800575-7a211500-88a2-11eb-9abd-b934c67964e3.png)
![Screenshot from 2021-03-19 11-10-29](https://user-images.githubusercontent.com/6207644/111801885-c28d0280-88a3-11eb-9323-61e4a2fde6fa.png)
